File: ShoppingELF/ShoppingELF/Models/OrderModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace ShoppingELF.Models
{
    public class OrderModel
    {
        public int OrderID { get; set; }
        public int UserID { get; set; }
        public string productBrand { get; set; }
        public string ProductName { get; set; }
        public int productPrice { get; set; }
        public string productSize { get; set; }
        public string productPicture { get; set; }
        public int productQuantity { get; set; }
        public int PID { get; set; }

        public virtual SizeTable SizeTable { get; set; }
        public virtual UserTable UserTable { get; set; }
    }
}

File: Frontend/src/app/shared/order.model.ts
export class Order {
    
OrderId:number;
UserID:number;
productBrand:string;
ProductName:string;
productPrice:number;
productSize:string;
productPicture:string;
productQuantity:number;

constructor(
  productBrand:string, productName:string,productPrice:number,productSize:string,prouctPicture:string,productQuantity:number
){


 this.productBrand=productBrand;
 this.ProductName=productName;
 this.productPrice=productPrice;
 this.productSize=productSize;
 this.productPicture=prouctPicture;
 this.productQuantity=productQuantity;


}




}